vault/resource_terraform_cloud_secret_backend.go

```diff
@@ -177,12 +210,12 @@ func terraformCloudSecretBackendUpdate(ctx context.Context, d *schema.ResourceDa
 
 	configPath := terraformCloudSecretBackendConfigPath(backend)
 
-	if d.HasChange("address") || d.HasChange("token") || d.HasChange("base_path") {
+	if d.HasChange(consts.FieldAddress) || d.HasChange(consts.FieldToken) || d.HasChange(consts.FieldBasePath) {
 		log.Printf("[DEBUG] Updating Terraform Cloud configuration at %q", configPath)
 		data := map[string]interface{}{
-			"address":   d.Get("address").(string),
-			"token":     d.Get("token").(string),
-			"base_path": d.Get("base_path").(string),
+			consts.FieldAddress:  d.Get(consts.FieldAddress).(string),
+			consts.FieldToken:    d.Get(consts.FieldToken).(string),
```

Comment:
I see that we always include `d.Get(consts.FieldToken)` when `address` or `base_path` changes. What happens if the resource was created with `token_wo` (so token is empty in state)? Will this overwrite the Vault config with an empty token?